bulk-delete.php

```diff
@@ -46,7 +46,6 @@
 	 * Version 6.1.0 of the Bulk Delete plugin dropped support for PHP < 5.2.
 	 * If you are still struck with PHP < 5.6 and can't update, then use v6.0.2 of the plugin.
 	 * But note that some add-ons may not work.
-	 *
```

Comment:
@esudalaimadan 

Any specific reason for making this change?

Otherwise the PR is good.

Reply:
@sudar StyleCI was failing. It was the change made by StyleCI.